adal/src/telemetry/java/com/microsoft/aad/adal/EventStrings.java

```diff
@@ -0,0 +1,163 @@
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package com.microsoft.aad.adal;
+
+final class EventStrings {
+
+    static final String EVENT_NAME = "event";
+
+    // Event names
+    static final String API_EVENT = "api_event";
+
+    static final String AUTHORITY_VALIDATION_EVENT = "authority_validation";
+
+    static final String HTTP_EVENT = "http_event";
+
+    static final String BROKER_EVENT = "broker_event";
+
+    static final String UI_EVENT = "ui_event";
+
+    static final String ENCRYPTION_EVENT = "encryption_event";
+
+    static final String DECRYPTION_EVENT = "decryption_event";
+
+    static final String TOKEN_CACHE_LOOKUP = "token_cache_lookup";
+
+    static final String TOKEN_CACHE_WRITE = "token_cache_write";
+
+    static final String TOKEN_CACHE_DELETE = "token_cache_delete";
+
+    // API ID
+    static final String ACQUIRE_TOKEN_SILENT_SYNC = "1";
+
+    static final String ACQUIRE_TOKEN_SILENT = "2";
+
+    static final String ACQUIRE_TOKEN_SILENT_ASYNC = "3";
+
+    static final String ACQUIRE_TOKEN_WITH_REFRESH_TOKEN = "4";
+
+    static final String ACQUIRE_TOKEN_WITH_REFRESH_TOKEN_2 = "5";
+
+    static final String ACQUIRE_TOKEN_1 = "100";
+
+    static final String ACQUIRE_TOKEN_2 = "104";
+
+    static final String ACQUIRE_TOKEN_3 = "108";
+
+    static final String ACQUIRE_TOKEN_4 = "111";
+
+    static final String ACQUIRE_TOKEN_5 = "115";
+
+    static final String ACQUIRE_TOKEN_6 = "116";
+
+    static final String ACQUIRE_TOKEN_7 = "117";
+
+    // Event Parameter strings
+    static final String APPLICATION_NAME = "application_name";
+
+    static final String APPLICATION_VERSION = "application_version";
+
+    static final String CLIENT_ID = "client_id";
+
+    static final String CLIENT_IP = "client_ip";
+
+    static final String AUTHORITY_NAME = "authority";
+
+    static final String AUTHORITY_TYPE = "authority_type";
+
+    static final String AUTHORITY_TYPE_ADFS = "adfs";
+
+    static final String AUTHORITY_TYPE_AAD = "aad";
+
+    static final String API_DEPRECATED = "is_deprecated";
```

Comment:
then add a doc please, trust me, you will probably don't remember what it means after couple of months. :)

---
In reply to: [76171588](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/703#discussion_r76171588) [](ancestors = 76171588,76171141)